seqr/urls.py

```diff
@@ -335,11 +335,16 @@
 urlpatterns += [
     url(r'^admin/login/$', RedirectView.as_view(url=LOGIN_URL, permanent=True, query_string=True)),
     url(r'^admin/', admin.site.urls),
-    url(r'^media/(?P<path>.*)$', django.views.static.serve, {
-        'document_root': MEDIA_ROOT,
-    }),
 ]
 
+# If serving /media from the local filesystem
```

Comment:
this change really confused me the first time I read it. Maybe add to the comment something like "not needed if using GCS as the media backend"

Reply:
Ok, I see, I can update the comment. The logic here is that we only want to serve requests to /media/* if we're configured to serve static content from a local media_root directory.

Reply:
resolved in #2397